client.go

```diff
@@ -30,20 +30,18 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
 // DAMAGE.
 
-
 package pilosa
 
 import (
 	"bytes"
 	"encoding/json"
 	"fmt"
+	"github.com/golang/protobuf/proto"
+	"github.com/pilosa/go-pilosa/internal"
```

Comment:
I was testing Gogland instead of VSCode (which runs goimports automatically). It seems Gogland didn't organize the imports. I'll run goimports manually and commit.